docs/dev/plugins/cloudformation/cloudformation-basics.md

```diff
@@ -0,0 +1,36 @@
+# AWS CloudFormation
+How to transform a TOSCA model to an Amazon Web Services CloudFormation template.
+
+## Mapping
+see [this](mapping.md)
+
+## Transformation Lyfecycle
+
+### validate
+
```

Comment:
You could add `nothing to do here so far` or something like that, so everybody knows that you didnt miss that.